File: server/Audi/Helpers/OrderParams.cs
namespace Audi.Helpers
{
    public class OrderParams : PaginationParams
    {
        public int? UserId { get; set; }
        public int? PriceMin { get; set; }
        public int? PriceMax { get; set; }
        public string OrderNumber { get; set; }
        public string CurrentStatus { get; set; }
        public string LastName { get; set; }
        public string FirstName { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
    }
}

File: client/projects/data/src/lib/models/order-params.ts
import { OrderStatus } from '../enums';
import { PagedRequest } from './pagination';

export interface OrderParams extends PagedRequest {
  userId?: number;
  priceMin?: number;
  priceMax?: number;
  orderNumber?: string;
  currentStatus?: OrderStatus;
  lastName?: string;
  firstName?: string;
  email?: string;
  phoneNumber?: string;
}
